pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -475,9 +475,8 @@ type LinuxSandboxSecurityContext struct {
 	NamespaceOptions *NamespaceOption `protobuf:"bytes,1,opt,name=namespace_options,json=namespaceOptions" json:"namespace_options,omitempty"`
 	// Optional SELinux context to be applied.
 	SelinuxOptions *SELinuxOption `protobuf:"bytes,2,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
-	// User to run the entrypoint of the sandbox process. Can be either UID or
-	// user name.
-	RunAsUser *string `protobuf:"bytes,3,opt,name=run_as_user,json=runAsUser" json:"run_as_user,omitempty"`
+	// UID to run sandbox processes as, when applicable.
```

Comment:
> Ideally, how the sandbox runs should totally be an implementation detail of the CRI runtime, and it should always run as securely as it can (that is without root whenever possible)... but the docker runtime today can have different pause user vs container user, and so it gets passed down.

It depends, running containers with elevated permissions via DaemonSets that expect to be able to monitor/see/update host-level things on the nodes means exposing some control over users in the API.